halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/ui/v1/ProblemSetWriter.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2016 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.halyard.cli.ui.v1;
+
+import com.netflix.spinnaker.halyard.config.model.v1.problem.Problem;
+import com.netflix.spinnaker.halyard.config.model.v1.problem.Problem.Severity;
+import com.netflix.spinnaker.halyard.config.model.v1.problem.ProblemSet;
+
+public class ProblemSetWriter {
```

Comment:
This was only being used by the ResponseUnwrapper, so I called the write method `formatProblemSet` and folded it in there.